### PR TITLE
Modal runner

### PR DESCRIPTION
**Pull request by Eliaxie**

### CODE REVIEW

**Comment by ripa1995** [comment]
@Eliaxie @bonomip nothing is working in this version.
It redirects directly to the landing page, then i can move only the slider, but i can't click on any button (play riddles, submit email, likedin/github).

**Comment by ripa1995** [comment]
Managed to test it. I've removed an extra div that was blocking the interaction. There is another issue, when closing the modal there is the animation that brings the landing page from the bottom of the screen (like when landing for the first time from the initial riddles).

File: src/components/MainSite.astro [hunk 1]
```diff
@@ -11,7 +12,16 @@ export interface Props {
...
+<div class="w-full m-auto">
+	<div class="absolute w-full h-full">
+		<div
+			class="middle hidden dark:block absolute animate-wigglemiddle <md:hidden left-[50%] h-full min-h-full cursor-ew-resize z-20">
+			<div
+				class="fixed transform -translate-x-2/4 w-10 h-10 border-1 bg-white border-[#FFAD8A] top-11/36 grid grid-cols-[1fr_1fr] rounded-1/2 items-center">
+				<img src={Short_Logo} class="ml-1/2 pointer-events-none" />
+			</div>
+		</div>
+	</div>
```
Comment (ripa1995) [inline src/components/MainSite.astro:24]: Apparently the issue is with this div, removing it makes the buttons work again.

Comment (Eliaxie) [reply src/components/MainSite.astro]: yep already removed, I've put it there by mistake

**Comment by Eliaxie** [comment]
regarding the animation issue, we know it's there, we need to go over the animations again
I need this pr merged into dev to go on with the restyling

**Review by ripa1995 — APPROVED** [review]

**Pull request merged**